content/en/md2html-setup.md

```diff
@@ -48,3 +48,44 @@ Example of content inside of a `index.md`
     * File name without extension format. See example above
 * `files`
     * This property, list all the names of the markdown documents to display in the order in which should appear on the final document
```

Comment:
Suggestion: This property represents a list of documents in the markdown that needs to be combined to create a final document.